Test/ProductsTests.cs

```diff
@@ -1,20 +1,30 @@
 ﻿using TMS_Selenium.Utils;
+using Allure.NUnit.Attributes;
+using Allure.Net.Commons;
 
 namespace TMS_Selenium.Test
 {
     public class ProductsTests : BaseTest
     {
         [Test]
+        [AllureDescription("Test to check Logout")]
+        [AllureSeverity(SeverityLevel.critical)]
+        [AllureIssue("LgO-01")]
+        [AllureFeature("Basic Functionality")]
+        [AllureStory("Logout")]
         public void Logout()
         {
             loginPage.Login(Configurator.ReadConfiguration().UserNameSauceDemo, Configurator.ReadConfiguration().PasswordSauceDemo);
-            productsPage.MenuButton().Click();
-            productsPage.LogoutLink().Click();
+            headerPage.Logout();
 
-            Assert.That(loginPage.LoginTitle().Displayed, "Login title is not displayed.");
+            Assert.That(loginPage.LoginTitle().Displayed);
         }
 
         [Test]
+        [AllureDescription("Test to check the displayed number of products in the cart and the text on the buttons after adding products to the cart")]
+        [AllureSeverity(SeverityLevel.critical)]

```

Comment:
You can start description without "Test to"
example: Check the displayed number of...